article/app/controllers/ArticleController.scala

```diff
@@ -104,42 +111,50 @@ object ArticleController extends Controller with RendersItemResponse with Loggin
       renderFormat(htmlResponse, jsonResponse, article, Switches.all)
   }
 
-  def renderArticle(path: String, lastUpdate: Option[String], rendered: Option[Boolean]) = {
-    if (LongCacheSwitch.isSwitchedOn) Action.async { implicit request =>
-      // we cannot sensibly decache memcached (does not support surogate keys)
-      // so if we are doing the 'soft purge' don't memcache
-      loadArticle(path, lastUpdate, rendered)
-    } else MemcachedAction { implicit request =>
-      loadArticle(path, lastUpdate, rendered)
+  def renderLive(path: String, page: Option[Int] = None) =
```

Comment:
I don't think you'll need the page parameter here, in future the page will be inferred from `request` using `inferPage`.